apps/webpage/src/app/archive/archive.component.ts

```diff
@@ -0,0 +1,222 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+interface Talk {
+  title: string;
+  author: string;
+  slidesUrl?: string;
+  description?: string;
+}
+
+export interface Meetup {
+  date: string;
+  location: string;
+  talks?: Talk[];
+}
+
+@Component({
+  selector: 'ng-stuttgart-archive',
+  templateUrl: './archive.component.html',
+  styleUrls: ['./archive.component.scss']
+})
+export class ArchiveComponent implements OnInit {
+  meetupIndex = 0;
+  selectedMeetup$: Observable<Meetup>;
+  meetups$: Observable<Meetup[]>;
+
+  selectMeetup(index: number) {
+    this.meetupIndex = index;
+    this.selectedMeetup$ = this.meetups$.pipe(
+      map((meetups: Meetup[]) => meetups[index])
+    );
+  }
+
+  constructor() {}
+
+  ngOnInit() {
+    this.meetups$ = of([
```

Comment:
BehaviourSubject?

Reply:
ich lass das erstmal so, das kommt bald in die nest app

Reply:
ok